src/lib/db/project-store.ts

```diff
@@ -238,7 +238,15 @@ class ProjectStore implements IProjectStore {
             .where({
                 project_id: id,
             })
-            .pluck('environment_name');
+            .innerJoin(
+                'environments',
+                'project_environments.environment_name',
+                'environments.name',
+            )
+            .orderBy('environments.sort_order', 'asc')
+            .orderBy('project_environments.environment_name', 'asc')
```

Comment:
Won't this second orderBy override the orderBy on sort order?

Reply:
My timing is terrible but I just found the answer - this is Knex's way of sorting on multiple columns: https://github.com/knex/knex/issues/2874